src/css/styles/_top.scss

```diff
@@ -1,5 +1,10 @@
 .top {
   position: relative;
+  width: 100%;
+  min-height: 500px;
+  overflow: hidden;
+  background: url("../img/banner-bg.png") no-repeat 75%;
```

Comment:
In future we would need switch too looping video, but it should not be so hard.
Ref: https://slicejack.com/fullscreen-html5-video-background-css/